.packit/rpmlint.spec

```diff
@@ -0,0 +1,88 @@
+%{!?python3: %global python3 %{__python3}}
+
+%bcond_with tests
+
+Name:           rpmlint
+Version:        2.0.0
+Release:        0%{?dist}
+Summary:        Tool for checking common errors in RPM packages
+
+License:        GPLv2+
+URL:            https://github.com/rpm-software-management/rpmlint
+Source0:        %{url}/archive/%{name}-%{version}.tar.gz
+
+BuildArch:      noarch
+
+BuildRequires:  python3-devel
```

Comment:
why? devel is needed only for FFI libraries

Reply:
Because this works everywhere. In Fedora and Mageia, you're supposed to BR `python3-devel` and get the macros and such that way. This also happens to work on openSUSE.